01_fib/01_fib.go

```diff
@@ -0,0 +1,30 @@
+
+package main
+
+import (
+"fmt"
+"io"
+"os"
+)
+
+var out io.Writer = os.Stdout
+
+var (
+	num1 = 0
+	num2 = 1
+	sum = 0
+)
+
+func main() {
+	Fib(5)
+	}
+		
+func Fib(n int) int {
+	for i := 0; i < n; i++ {
+	 	sum = num1 + num2
+        	num1 = num2
+        	num2 = sum
```

Comment:
There's a mix of spaces and tabs here. Again, the Go extension should autocorrect this. But you should also rely on <kbd>Tab</kbd> to do this for you. Don't hammer the <kbd>space</kbd> bar.

Reply:
Done